### PR TITLE
site: Add Contour v1.9.0 blog post along with demo yaml files

### LINKED ISSUE #2934: Tracking issue: Blog post for 1.9 release
This is a tracking issue (as discussed in the community meeting) to ensure we have a blog going out together with the 1.9 release, explaining new features and functionality.

- [x] Assigned blog author
- [x] Blog drafted
- [ ] Blog reviewed
- [ ] Blog published

### PR DESCRIPTION
**Pull request by stevesloka**
Signed-off-by: Steve Sloka <slokas@vmware.com>

Closes #2934

### CODE REVIEW

File: site/_posts/2020-10-07-contour_v190.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+CRDs became [generally available in Kubernetes 1.16](https://kubernetes.io/blog/2019/09/18/kubernetes-1-16-release-announcement/#custom-resources-reach-general-availability) over a year ago.
+
+This change bumps Contour’s minimum supported Kubernetes version to 1.16.
+
+## Join the Contour Community
+
+* Join the [Contour Community Meetings](https://projectcontour.io/community/) or [Office Hours](https://github.com/projectcontour/community/wiki/Office-Hours)
+* Get updates on Twitter ([@projectcontour](https://twitter.com/projectcontour0))
+* Chat with us on Slack ([#contour](https://kubernetes.slack.com/messages/C8XRH2R4J) on Kubernetes Slack)
+* Collaborate with us on GitHub: [github.com/projectcontour/contour](https://github.com/projectcontour/contour)
```
Comment (jonasrosland) [inline site/_posts/2020-10-07-contour_v190.md:99]: You can remove this, we have a partial layout that covers the community stuff in each blog post: https://github.com/projectcontour/contour/blob/main/site/_includes/blog-community-footer.html

File: site/_posts/2020-10-07-contour_v190.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+We’re immensely grateful for all the community contributions that help make Contour even better! For version 1.9, special thanks go out to the following people:
+- @aberasarte
+- @bgagnon
+- @glerchundi
+- @mattmoor
+- @ShaileshSurya
+- @tong101
+- @tsaarni
+- @zianke
```
Comment (jonasrosland) [inline site/_posts/2020-10-07-contour_v190.md:109]: Please add links to the GitHub profiles for the contributors

File: site/_posts/2020-10-07-contour_v190.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+### Sequence of Events for External Auth:
+1. Deploy an External Authorization Service to your cluster: This service talks to your Authorization Provider and determines if the request should be authorized or not.
+2. Create an `ExtensionService` CRD: This CRD allows the External Authorization Service created in the previous step to be available so that Contour can configure Envoy with that gRPC endpoint.
+3. Create HTTPProxy resource: The VirtualHost in the ingress object references the ExternalService CRD linking that virtual host to the authorization service. 
+4. On every client request, Envoy sends an authorization check  to the External Auth Service to determine authorization.
+
+## Demo
+Here's a quick video demonstration walking through the setup and configuration of External Authorization with Contour.
+
+[![img][2]][3]
```
Comment (jonasrosland) [inline site/_posts/2020-10-07-contour_v190.md:30]: You should be able to embed the YouTube video through this: https://github.com/projectcontour/contour/blob/main/site/_plugins/youtube.rb

If you replace this with `{% youtube wm_eWO4mZYs %}` it should embed the video.

File: site/_posts/2020-10-07-contour_v190.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+- @glerchundi
+- @mattmoor
+- @ShaileshSurya
+- @tong101
+- @tsaarni
+- @zianke
+
+[1]: {% link img/posts/contourauth.png %}
+[2]: {% link img/posts/authdemo.png %}
+[3]: https://youtu.be/wm_eWO4mZYs
```
Comment (jonasrosland) [inline site/_posts/2020-10-07-contour_v190.md:113]: If we embed the video this won't be needed, and `site/img/posts/authdemo.png` can be removed too.

**Review by jonasrosland — CHANGES_REQUESTED** [review]

**Comment by stevesloka** [comment]
Comments addressed @jonasrosland!

File: site/_posts/2020-10-07-contour_v190.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+## Community Thanks!
+We’re immensely grateful for all the community contributions that help make Contour even better! For version 1.9, special thanks go out to the following contributors:
+- [@aberasarte](https://github.com/aberasarte)
+- [@bgagnon](https://github.com/bgagnon)
+- [@glerchundi](https://github.com/glerchundi)
+- [@mattmoor](https://github.com/mattmoor)
+- [@ShaileshSurya](https://github.com/ShaileshSurya)
+- [@tong101](https://github.com/tong101)
+- [@tong101](https://github.com/tong101)
```
Comment (jonasrosland) [inline site/_posts/2020-10-07-contour_v190.md:102]: Duplicate contributor :)

**Review by jonasrosland — CHANGES_REQUESTED** [review]

File: site/_posts/2020-10-07-contour_v190.md (hunk 5 above)
Comment (stevesloka) [reply site/_posts/2020-10-07-contour_v190.md]: 😞

**Review by jonasrosland — APPROVED** [review]

**Pull request merged**